Source/JobGiver_WanderZombieHerd.cs

```diff
@@ -71,11 +67,8 @@ protected override Job TryGiveJob(Pawn pawn)
                             {
                                 randomCell = pawnPath.NodesReversed[pawnPath.NodesReversed.Count - wanderLength];
 
-                                
-                                return new Job(ZombiefiedMod.zombieMove, randomCell)
-                                {
-                                    //expiryInterval = 777
```

Comment:
For example here how do we remember that it is possible to set an expiryInterval now?

Reply:
🤔 Well since it's a part of the `Job `class I'd argue some documentation somewhere should exist for that class to say it's possible. 

I feel it'd be consistency is key with this sort of thing. There's not a comment for the 20 or so other variables that could be set for jobs, so why this one? I think documenting Rimworld's Job stuff would be suuuuper useful, but I don't think it should be done in comments in an arbitrary usage of it.

If it's particularly useful though, rather than just having code without much context, maybe a sentence or two to explain what it does and why it might need to be defined?

Reply:
Another reason I might avoid these types of comments is that this type of comment is subject to change when Rimworld updates, and if these comments stick around through updates, it becomes difficult to figure out what's relevant and what isn't, so a majority of the time you'd wind up checking decomplication anyway to figure out what to do. 

At least with some kind of sentence explaining it and the usage would convey the intent, making it future proof to, I donno, random variable name changes or something.

Reply:
Did some digging in the decompilation, and I'm not even sure `expiryInterval` even does anything. It is defined in Jobs, but it doesn't seem like it's actually used for anything. This may be a relic of the past after all. 👁️ 👄 👁️

Reply:
There is a comment for this one and not others because this one was used here before and it is still relevant. Its possible that I would read this code and uncomment that line to add the expiry back again. If it really is deprecated that sucks ofcourse

Reply:
Also Im not sure what this emoji 👁️ 👄 👁️ means but it looks funny

Reply:
No I wont avoid leaving random bits of old code as comments thanks tho